kubernetes-model-generator/kubernetes-model-common/src/main/java/io/fabric8/kubernetes/model/annotation/Group.java

```diff
@@ -27,9 +27,10 @@
  * determine the `apiVersion` field associated with the annotated resource.
  * See https://kubernetes.io/docs/reference/using-api/#api-groups for more details.
  */
-@Target({TYPE})
+@Target({ TYPE })
 @Retention(RUNTIME)
 public @interface Group {
+  String NAME = Group.class.getSimpleName();
```

Comment:
🤓 I'm curious about this constants. Does this provide performance improvements? what's the reasoning?

Reply:
Can probably be removed indeed.